### PR TITLE
Working PR - Restructure accessibilty SPEC

### PR DESCRIPTION
**Pull request by isabela-pf**
A working PR for @MarsBarLee and I to review restructuring this SPEC and adding relevant content before opening a PR to the upstream repo.

### CODE REVIEW

File: spec-0003/index.md [hunk 1]
```diff
@@ -17,37 +17,46 @@ If relevant, include examples of how the new functionality would be used,
...
-She used Braille to communicate with others who were Deaf-Blind.
-It also allowed her to publish her experiences [citation for Story of My Life] and advocate for the Deaf-Blind community to the general public.
+## Community
 
-Accessibility benefits everyone.
-For example, captions for videos benefit not only Deaf viewers with complete hearing loss, but also viewers with partial hearing loss due to age.
-Captions also benefit viewers who cannot play audio out loud, such as in a quiet environment or if they have broken speakers.
-[citation to range of disabilities and situational disability].
-Thus accessibility benefits not just “a few disabled people”[a] but all people.
+At the time of writing, accessibility is not a regular consideration in many Scientific Python projects. Many projects may have a handful of people who have accessibility knowledge or are interested in accquiring more, but may not be able to reach critical mass on a per-project basis. Connecting Scientific Python community members focused on accessibility will provide a boon for individuals, projects, and the ecosystem at large.
```
Comment (MarsBarLee) [inline spec-0003/index.md]: Later once the url is live, we can link to accessibility.scientific-python.org/community.html page, which is the community.rst page

Comment (MarsBarLee) [reply spec-0003/index.md]:
```suggestion
At the time of writing, accessibility is not a regular consideration in many Scientific Python projects. Many projects may have a handful of people who have accessibility knowledge or are interested in acquiring more, but may not be able to reach critical mass on a per-project basis. Connecting Scientific Python community members focused on accessibility will provide a boon for individuals, projects, and the ecosystem at large. 

If you are interested in this effort, [join us now](accessibility.scientific-python.org/community.html)!
```
Added the 'call to action' and link.

File: spec-0003/index.md [hunk 2]
```diff
@@ -17,37 +17,46 @@ If relevant, include examples of how the new functionality would be used,
...
+There are accessibility guidelines that extend beyond the Scientific Python ecosystem; we respect those first and foremost. This includes
+
+- [Web Content Accessibility Guidelines (W3C)](https://www.w3.org/TR/WCAG/)
+
+Please note that in many cases guidelines serve as a foundation for accessibility efforts. They are by no means the limit of what is needed to create enjoyably accessible work.
+
+### Scientific Python recommendations
+
+For cases where ecosystem-specific recommendations are needed or examples of accessibility within the ecosystem are helpful, we also provide a list of resources designed to support the needs of our community.
+
```
Comment (MarsBarLee) [inline spec-0003/index.md:52]:
```suggestion

If you would like to add, edit or suggest new recommendations, join the community at [accessibility.scientific-python.org](accessibility.scientific-python.org/community.html)
```

**Comment by isabela-pf** [comment]
Closing this to make a PR upstream. Thank you for the feedback!

**Comment by isabela-pf** [comment]
Upstream PR is at [scientific-python/specs #134](https://github.com/scientific-python/specs/pull/134)

**Pull request closed without merging**